ruby/ext/google/protobuf_c/message.c

```diff
@@ -63,9 +63,13 @@ static void Message_mark(void* _self) {
   rb_gc_mark(self->arena);
 }
 
+static size_t Message_memsize(void* _self) {
+  return sizeof(Message);
```

Comment:
Same here, I'm assuming:

```
  const upb_Message* msg;  // Can get as mutable when non-frozen.
  const upb_MessageDef*
```

Are both pointers into Arena.

Reply:
The message, yes. The MessageDef is a pointer into the `upb_DefPool` (underlying the `DescriptorPool` class).

Reply:
Right, so I shouldn't consider it either since it should be reported by the `DescriptorPool` `memsize` function.